tests/e2escenarios/e2e_devfile_test.go

```diff
@@ -71,9 +71,10 @@ var _ = Describe("odo devfile supported tests", func() {
 		It("Verify output debug information for java-springboot works", func() {
 			createStarterProjAndSetDebug("java-springboot", "springbootproject")
 		})
-		It("Verify output debug information for java-quarkus debug works", func() {
-			createStarterProjAndSetDebug("java-quarkus", "community")
-		})
+		// enable it once https://github.com/redhat-developer/odo/issues/6181 is fixed
+		// It("Verify output debug information for java-quarkus debug works", func() {
```

Comment:
nit: This is fine as a quick fix, but I would suggest using Ginkgo's `Skip` function, so the test can still be visible as skipped in the reports.